src/Sulu/Bundle/ContentBundle/DependencyInjection/Configuration.php

```diff
@@ -52,6 +52,15 @@ public function getConfigTreeBuilder()
                         ->booleanNode('websocket')->defaultFalse()->end()
                     ->end()
                 ->end()
+                ->arrayNode('templates')
```

Comment:
What about calling this `default_templates`? I think it is not more than a default, it could be changed later.